lib/network_manager.go

```diff
@@ -209,9 +209,11 @@ func (nm *NetworkManager) startNonValidatorConnector() {
 			nm.exitGroup.Done()
 			return
 		case <-time.After(nm.peerConnectionRefreshInterval):
+			glog.V(2).Infof("NetworkManager.startNonValidatorConnector: Refreshing NON validator indices...")
```

Comment:
Sorry delete this and the one below it as well. Don't want to have any logging in this select.